runners/katacoda/index.ts

```diff
@@ -154,6 +153,7 @@ export class Katacoda extends Runner {
             "title": "Build the java project",
             "text": "step" + this.stepsCount + ".md"
         });
+        
```

Comment:
The second parameter is optional, so you must first check that the second parameter is set (default is false).
In addition, the second parameter is an indicator of whether the tests should be run. So if the parameter is true, 'maven.test.skip' has to be false. And vice versa.